src/data/actions/grades.js

```diff
@@ -21,7 +21,22 @@ const fetchGrades = courseId => (
         dispatch(finishedFetchingGrades());
       })
       .catch((error) => {
-        dispatch(errorFetchingGrades())
+        dispatch(errorFetchingGrades());
+      });
+  }
+);
+
+const fetchUserGrades = (courseId, searchText) => (
```

Comment:
I think this action can be `fetchMatchingUserGrades`

Reply:
I agree, that is much better. I didn't know what to name it. Thanks!